internal/monitor/bucket.go

```diff
@@ -203,6 +204,16 @@ func (mb *monitoringBucket) DeleteFolder(ctx context.Context, folderName string)
 	return err
 }
 
+func (mb *monitoringBucket) RenameFolder(
+	ctx context.Context,
```

Comment:
nit: Dont add each function parameter in new line:

go/go-style/decisions#func-formatting

Fix it everywhere.